src/styles/app/30-markdown-editor-pane.css

```
@@ -95,65 +95,53 @@
 }
 
 .markdownEditorModeSwitch {
-	display: inline-grid;
-	grid-template-columns: repeat(3, minmax(0, auto));
+	--markdown-editor-mode-bubble-offset: 6px;
+	position: relative;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)